echidna/core/smear.py

```diff
@@ -233,6 +237,81 @@ def random_gaussian_energy_spectra(self, true_spectrum):
         smeared_spectrum._raw_events = raw_events
         return smeared_spectrum
 
+
+    def random_gaussian_energy_spectra_parallel(self, true_spectrum,nProcs):
+        """ Smears the energy of a spectra object using multiprocessing  by generating
+          a number of random points from a Gaussian pdf generated
+          for that bin. The number of points generated is equivalent
+          to the number of entries in that bin.
+
+        Args:
+          true_spectrum (spectra): spectrum to be smeared
+
+        Returns:
+          A smeared spectra object.
+        """
+        def Worker(true_spectrum,lower,upper,out_q):
+            smeared_chunk = spectra.Spectra(
+                true_spectrum._name+str(self._light_yield)+"_light_yield",
+                true_spectrum._num_decays)
+            raw_events = true_spectrum._raw_events
+            energy_step = (true_spectrum._energy_high-true_spectrum._energy_low)/true_spectrum._energy_bins
+            time_step = (true_spectrum._time_high-true_spectrum._time_low)/true_spectrum._time_bins
+            radial_step = (true_spectrum._radial_high-true_spectrum._radial_low)/true_spectrum._radial_bins
+            for time_bin in range(int(lower),int(upper)):
```

Comment:
Why split the `time` axis and not the `energy` axis, which is the one we are smearing?